src/user/Kelvin_initialization.F90

```diff
@@ -128,8 +128,8 @@ function register_Kelvin_OBC(param_file, CS, US, OBC_Reg)
   call register_OBC(casename, param_file, OBC_Reg)
   register_Kelvin_OBC = .true.
 
-  if (CS%mode > 0) call MOM_error(WARNING, &
```

Comment:
I agree with all of the other changes in this PR, but I think that instead of commenting out this warning message, it should perhaps be eliminated altogether.